sdk/src/wallet/storage/constants.rs

```diff
@@ -15,18 +15,15 @@ pub const fn default_storage_path() -> &'static str {
     DEFAULT_STORAGE_PATH
 }
 
-pub(crate) const WALLET_INDEXATION_KEY: &str = "iota-wallet-account-manager";
-
-pub(crate) const SECRET_MANAGER_KEY: &str = "secret_manager";
-
-pub(crate) const ACCOUNTS_INDEXATION_KEY: &str = "iota-wallet-accounts";
-pub(crate) const ACCOUNT_INDEXATION_KEY: &str = "iota-wallet-account-";
-
-pub(crate) const ACCOUNT_SYNC_OPTIONS: &str = "sync-options";
-
 pub(crate) const DATABASE_SCHEMA_VERSION: u8 = 1;
 pub(crate) const DATABASE_SCHEMA_VERSION_KEY: &str = "database-schema-version";
 
+pub(crate) const WALLET_DATA_KEY: &str = "wallet-data";
+pub(crate) const WALLET_BUILDER_KEY: &str = "wallet-builder";
```

Comment:
little weird, maybe we should call this something else?

Reply:
Honestly I'd keep these questions for later, we have no clue what the database with look like in the future. Adding it to https://github.com/iotaledger/iota-sdk/issues/1521

Reply:
alrighty